src/index.ts

```diff
@@ -231,6 +233,70 @@ function launchMetals(
       );
     }
 
+    registerCommand("metals.new-scala-file", async () => {
+      const currentDoc = await workspace.document;
+      const currentPath = currentDoc.uri;
+      const parentDir = path.dirname(currentPath);
+
+      const fileOptions = [
```

Comment:
Originally I figured we'd just share this with the VS Code extension, but annoyingly, coc.nvim only returns the number of the selection, so I have to have some sort of lookup. I won't leave this here though.

Reply:
I have an idea for a generic `quickPick` API for metals-languageclient, based on poorman typeclasses. For now it's fine as is.

Also, `value` can just be the array index (but maybe you wanted to be explicit)

Reply:
I wanted to be explicit... _lets go with that_

Kidding aside, it totally slipped my mind. Good idea. 👍